ci-operator/config/openshift/operator-framework-catalogd/openshift-operator-framework-catalogd-release-4.15.yaml

```diff
@@ -27,8 +27,6 @@ images:
   to: test-catalog
 promotion:
   disabled: true
```

Comment:
I saw that this was added in [this PR](https://github.com/openshift/release/commit/8dc1acba96c0c17f9b5bd060a4a7f8c14595f0bb), I suspect that this is supposed to be present on y stream releases that haven't GA'd.